239SlidingWindowMaximum.py

```
@@ -0,0 +1,25 @@
+class Solution:
+    def maxSlidingWindow(self, nums: list[int], k: int) -> list[int]:
+        stack=[]
+        n=len(nums)
+        for i in range(n-k+1):
+            stack.append(max(nums[i:i+k]))
+        return stack 
+    
+class Solution:
```

Ruff (0.12.2)

Redefinition of unused `Solution` from line 1

(F811)